recipes/libnl/libnl_3.2.27.oe

```diff
@@ -2,9 +2,10 @@ require ${PN}.inc
 
 LICENSE = "LGPLv2.1+"
 
+SRC_URI = "https://github.com/thom311/libnl/releases/download/libnl3_2_27/libnl-${PV}.tar.gz"
```

Comment:
We need some kind of feature to parse ${PV} from dots to underscores. Almost all github projects release with these paths